resolwe/flow/tests/test_commands.py

```diff
@@ -88,3 +91,53 @@ def test_process_register_no_admin(self):
         err = StringIO()
         self.assertRaises(SystemExit, call_command, 'register', path=[PROCESSES_DIR], stderr=err)
         self.assertEqual('Admin does not exist: create a superuser\n', err.getvalue())
+
+
+class ListDockerImagesTest(ProcessTestCase):
```

Comment:
Domen told me to use `_register_schemas` instead of calling the `register` command, since it's faster.